packages/sitecore-jss/src/utils/resolve-url.ts

```diff
@@ -47,4 +47,16 @@ function resolveUrl(urlBase: string, params: ParsedUrlQueryInput = {}): string {
   return result;
 }
 
+export const isAbsoluteUrl = (url: string) => {
```

Comment:
This seems misplaced in this file. Might be better to add new file, or rename this one to something more generic without the default export.

Reply:
It's a good point, I decided to rename file, it's more clear